frontend/src/features/admin-form/settings/components/AuthSettingsSection/AuthSettingsSection.tsx

```diff
@@ -1,181 +1,61 @@
-import {
-  Fragment,
-  KeyboardEventHandler,
-  MouseEventHandler,
-  useCallback,
-  useMemo,
-  useState,
-} from 'react'
-import { Box, Flex, Skeleton, Spacer, Text } from '@chakra-ui/react'
+import { useMemo } from 'react'
+import { Box, Skeleton } from '@chakra-ui/react'
 
 import { FormAuthType, FormSettings, FormStatus } from '~shared/types/form'
 
-import { GUIDE_SPCP_ESRVCID } from '~constants/links'
-import InlineMessage from '~components/InlineMessage'
-import Link from '~components/Link'
-import Radio from '~components/Radio'
-import { Tag } from '~components/Tag'
+import Toggle from '~components/Toggle'
 
 import { useAdminForm } from '~features/admin-form/common/queries'
 import { isMyInfo } from '~features/myinfo/utils'
 
-import { useMutateFormSettings } from '../../mutations'
-
-import { FORM_AUTHTYPES } from './constants'
-import { EsrvcIdBox } from './EsrvcIdBox'
-
-const esrvcidRequired = (authType: FormAuthType) => {
-  switch (authType) {
-    case FormAuthType.SP:
-    case FormAuthType.MyInfo:
-    case FormAuthType.CP:
-      return true
-    default:
-      return false
-  }
-}
+import { AuthSettingsDescriptionText } from './AuthSettingsDescriptionText'
+import { AuthSettingsDisabledExplanationText } from './AuthSettingsDisabledExplanationText'
+import { AuthSettingsSingpassSection } from './AuthSettingsSingpassSection'
+import { FormSingpassAuthToggle } from './FormSingpassAuthToggle'
 
 interface AuthSettingsSectionProps {
   settings: FormSettings
 }
 
 export const AuthSettingsSectionSkeleton = (): JSX.Element => {
   return (
-    <Radio.RadioGroup>
-      {Object.entries(FORM_AUTHTYPES).map(([authType, textToRender]) => (
-        <Radio isDisabled key={authType}>
-          <Skeleton>{textToRender}</Skeleton>
-        </Radio>
-      ))}
-    </Radio.RadioGroup>
+    <Skeleton>
+      <AuthSettingsDescriptionText />
+      <Toggle description="" label="" />
```

Comment:
Looks like description can be omitted (need to check if this affects the rendered output)

Reply:
verified rendered output that there are no visual changes. removed the description prop!